src/index.ts

```diff
@@ -134,17 +140,26 @@ export class I2C extends Peripheral {
     super.destroy();
   }
 
-  private getDevice(address: number) {
+  private getDevice(address: number, busNumber?: number) {
     let device = this.devices[address];
 
     if (device === undefined) {
-      device = openSync(getBoardRevision() === VERSION_1_MODEL_B_REV_1 ? 0 : 1);
+      if (busNumber === undefined) {
+        busNumber = this.busNumber;
+      }
+      device = openSync(busNumber);
       this.devices[address] = device;
     }
 
     return device;
   }
 
+  public configure(options?: any): void {
```

Comment:
Thanks for opening the issue @fivdi. I knew when i did that implementation that it would not work if i was reading the same address on both devices. But as it was something that needed deep fixing withing johnny-five, i kind of let it slide ...